### PR TITLE
[NFC] Update RVec rel notes

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

File: README/ReleaseNotes/v628/index.md [hunk 1]
```diff
@@ -228,6 +228,11 @@ The `ROOT::Math::KahanSum` class was slightly modified:
 
 The usage of `TRef` in the `TFoamCell` class has ben replaced with array indices. This avoids, when generating a large number of toys requiring a re-initialization of `TFoam` an increase in the memory usage caused by `TRef`.
 
+### RVec
+
+- a number of new helper functions have been added to [RVec](https://root.cern/doc/master/classROOT_1_1VecOps_1_1RVec.html): [Range](https://root.cern/doc/master/group__vecops.html#ga59cc6e477803f2bfd7dae29e56048cc1), [Product](https://root.cern/doc/master/group__vecops.html#ga25e4c2cf5c82fe56dd6bbc86b2386b69) and Enumerate
```
Comment (eguiraud) [inline README/ReleaseNotes/v628/index.md:233]: because of the problem fixed by the first commit in this PR we don't have a link for `Enumerate` atm

Comment (Axel-Naumann) [reply README/ReleaseNotes/v628/index.md:233]: Oh c'mon Doxygen 🙄 I'll try to remember to update the link before we release.

**Review by Axel-Naumann — APPROVED** [review]

**Comment by Axel-Naumann** [comment]
Let me merge, I want us to converge. And this looks good!

**Comment by Axel-Naumann** [comment]
Oh and this is master - I'll backport (without PR just push as it's NFC)

**Comment by Axel-Naumann** [comment]
07823c7e5f23ee53b5897084b85daa667ac53fdc in v6-28-00-patches.

**Pull request merged**